azure_li_services/units/system_setup.py

```diff
@@ -71,6 +74,53 @@ def set_energy_performance_settings():
     _write_boot_local(cpupower_calls)
 
 
+def set_kdump_service(high, low):
+    calibrated = _kdump_calibrate()
```

Comment:
This should only run if "high" or "low" are "None", if we have both values we do not need/want to run the tool

Reply:
yes, will change it